https://github.com/Plagman/gamescope/issues/316

../src/wlserver.cpp:26:10: fatal error: 'wlr/backend/noop.h' file not found
#include <wlr/backend/noop.h>
         ^~~~~~~~~~~~~~~~~~~~

--- src/wlserver.cpp.orig	2021-10-18 03:17:30 UTC
+++ src/wlserver.cpp
@@ -23,7 +23,6 @@ extern "C" {
 #include <wlr/backend/headless.h>
 #include <wlr/backend/multi.h>
 #include <wlr/backend/libinput.h>
-#include <wlr/backend/noop.h>
 #include <wlr/interfaces/wlr_input_device.h>
 #include <wlr/interfaces/wlr_keyboard.h>
 #include <wlr/render/wlr_renderer.h>
@@ -642,10 +641,10 @@ bool wlserver_init( void ) {
 
 	wl_signal_add( &wlserver.wlr.multi_backend->events.new_input, &new_input_listener );
 
-	wlserver.wlr.noop_backend = wlr_noop_backend_create( wlserver.display );
+	wlserver.wlr.noop_backend = wlr_headless_backend_create( wlserver.display );
 	wlr_multi_backend_add( wlserver.wlr.multi_backend, wlserver.wlr.noop_backend );
 
-	wlserver.wlr.output = wlr_noop_add_output( wlserver.wlr.noop_backend );
+	wlserver.wlr.output = wlr_headless_add_output( wlserver.wlr.noop_backend, 800, 600 );
 
 	if ( bIsDRM == True )
 	{
